Automattic/wp-calypso
# Signup: Don't show the login option in the masterbar on /login

On /signup we should hide the "Sign Up" link and on /login we should hide the "Log In" link:

<img width="901" alt="screen shot 2017-04-06 at 17 48 59" src="https://cloud.githubusercontent.com/assets/275961/24765871/5eae22ae-1af1-11e7-87b1-38b7e710f6c6.png">
<img width="899" alt="screen shot 2017-04-06 at 17 48 49" src="https://cloud.githubusercontent.com/assets/275961/24765872/5eaf8c66-1af1-11e7-881b-ab6764347566.png">

This used to happen but the change was lost at some point, so this reinstates the change for signup and adds it for login.
  
#### Testing instructions
  
1. Run `git checkout update/masterbar-logged-out` and start your server, or open a [live branch](https://delphin.live/?branch=update/masterbar-logged-out)
2. Open the [`Login` page](http://calypso.localhost:3000/login)
3. Check that there is no "Log in" link in masterbar
2. Open the [`Signup` page](http://calypso.localhost:3000/start)
3. Check that there is no "Log in" link in masterbar
  
#### Reviews
  
   
@Automattic/sdev-feed
